shared/common-adapters/rich-button.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import * as Sb from '../stories/storybook'
+
+import RichButton from './rich-button'
+
+const Kb = {
+  RichButton,
+}
+
+const load = () => {
+  Sb.storiesOf('Common/RichButton', module).add('Basic', () => (
+    <Kb.RichButton
+      onClick={() => {
+        console.log('hi')
```

Comment:
You can do `onClick={Sb.action('onClick')}` and it'll show up in the actions panel on desktop